op-chain-ops/genesis/config_test.go

```diff
@@ -191,10 +191,6 @@ func TestUpgradeScheduleDeployConfig_ActivateForkAtOffset(t *testing.T) {
 func TestUpgradeScheduleDeployConfig_SolidityForkNumber(t *testing.T) {
 	// Iterate over all of them in case more are added
 	for i, fork := range scheduleableForks[2:] {
-		if fork == "interop" {
-			continue
-		}
```

Comment:
Why did we used skip this test for interop, but not anymore? I see interop defined in the [`Fork` enum](https://github.com/ethereum-optimism/optimism/blob/b6a0ab982faea9301e27f4ecaaa4c589470cde0d/packages/contracts-bedrock/scripts/libraries/Config.sol#L31), but not in [`ForkUtils.toString`](https://github.com/ethereum-optimism/optimism/blob/b6a0ab982faea9301e27f4ecaaa4c589470cde0d/packages/contracts-bedrock/scripts/libraries/Config.sol#L46), unsure if that's related

Reply:
I think the logic was just wrong before. I believe that I was trying to avoid an array overflow error by stopping at interop (previously, the second last fork), but that doesn't really matter in this case because we're not using `i` as an array index anywhere.